lib/rubocop/cop/layout/space_inside_hash_literal_braces.rb

```diff
@@ -5,6 +5,13 @@ module Cop
     module Layout
       # Checks that braces used for hash literals have or don't have
       # surrounding space depending on configuration.
+      #
+      # @example
```

Comment:
As this cop is configurable it should actually have different examples for each supported code style. Just take a look at the documentation of something like https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/braces_around_hash_parameters.rb